lib/libp2p_port.ex

```diff
@@ -530,6 +539,49 @@ defmodule LambdaEthereumConsensus.Libp2pPort do
     {:noreply, state}
   end
 
+  @impl GenServer
+  def handle_call(:get_keystores, _from, %{validators: []} = state),
+    do: {:reply, [], state}
```

Comment:
Why is this clause necessary? If the validators list is empty, the enum.map will work, and return an empty list too.

Reply:
You are right! It is not necessary.